front/src/applications/stdcmV2/components/StdcmOrigin.tsx

```diff
@@ -50,10 +55,21 @@ const StdcmOrigin = ({
 
   const updateOriginPoint = (pathStep: PathStep | null) => {
     dispatch(updateOrigin(pathStep));
+    if (arrivalDateInput && arrivalTimeInput) {
+      const newOpArrival = generateISOArrival(arrivalDateInput, arrivalTimeInput);
+      dispatch(updateOriginArrival(newOpArrival));
+    }
```

Comment:
To avoid dispatching two actions when arrivalDate and arrivalTime are defined, what do you think about dispatching a single `updateOrigin(pathStep)` with the pathStep already having the new arrival time set?

The same logic can be applied for `updateDestinationPoint`

```suggestion
const updateOriginPoint = (pathStep: PathStep | null) => {
    if (!pathStep || !arrivalDateInput || !arrivalTimeInput) {
      return dispatch(updateOrigin(pathStep));
    }
    dispatch(
      updateOrigin({
        ...pathStep,
        arrival: generateISOArrival(arrivalDateInput, arrivalTimeInput),
      })
    );
  };
```